pihole/datadog_checks/pihole/data/conf.yaml.example

```diff
@@ -96,7 +96,7 @@ instances:
     ## a domain that is used in the check. For NTLM Auth, append
     ## the username to domain, not as the `username` parameter.
     #
-    # ntlm_domain: <NTLM_DOMAIN>/<USERNAME>
+    # ntlm_domain: <NTLM_DOMAIN>\<USERNAME>
```

Comment:
This looks like an unrelated change, I'll update this conf.yaml.example in a separate PR. You can remove these changes, thanks!

Reply:
reverted this patch